File: words/words.js
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout
});

readline.question(`Enter your text: `, text => {
  words = text.trim().split(" ");
  console.log(`Your text contains ${words.length} words.`);
  readline.close();
});

File: words/Words.scala
object Words {

    def countWords(str: String): Array[String] = {
        str.split(" ")
    }

    def main(args: Array[String]): Unit = {
        val input = scala.io.StdIn.readLine("Enter a sentence:")
        println(countWords(input).length)
    }
}